packages/contracts/hardhat.config.ts

```diff
@@ -37,7 +37,7 @@ const getCommonNetworkConfig = (networkName: ESupportedChains, chainId: number,
   saveDeployments: true,
   chainId,
   accounts: process.env.PRIVATE_KEY
-    ? [`0x${process.env.PRIVATE_KEY}`]
+    ? [process.env.PRIVATE_KEY]
```

Comment:
This is a good fix for the unset step in the coordinator workflow 🚀 . Thank you very much